kb/temporal-platform-limits-sheet.md

```diff
@@ -76,3 +76,6 @@ The Temporal Cloud service sets default limits for the following aspects:
     This means that you can't apply custom ordering of Workflows with Cloud Visibility features.
     Lists of Workflows are still ordered by a default ordering rule.
     Be aware that this rule might change.
+
+Some of the Temporal Cloud limits mentioned above can be increased by submitting a support ticket.
```

Comment:
Suggest linking "support ticket" to /cloud/how-to-create-a-ticket-for-temporal-support.